ParserHelpers.h

```diff
@@ -12,9 +12,9 @@ namespace commonItems
 // in the parser.
 constexpr char catchallRegex[] = R"([^=^{^}]+|\".+\")";
 
-void ignoreItem(const std::string& unused, std::istream& theStream);
-void ignoreObject(const std::string& unused, std::istream& theStream);
-void ignoreString(const std::string& unused, std::istream& theStream);
+void ignoreItem(std::istream& theStream);
```

Comment:
For the moment, keep both versions, but mark the ones with the unused parameter as deprecated.

Reply:
Which does mean keeping the old tests, too.

Reply:
After reverting the ignoreItem changes this is no longer needed.